src/main/java/io/vertx/core/spi/cluster/ClusterManager.java

```diff
@@ -44,31 +41,26 @@
  */
 public interface ClusterManager {
 
-  void setVertx(Vertx vertx);
-
-  /**
-   * Return an async multi-map for the given name
-   */
-  <K, V> void getAsyncMultiMap(String name, Handler<AsyncResult<AsyncMultiMap<K, V>>> resultHandler);
+  void init(Vertx vertx, DeliveryStrategy deliveryStrategy);
```

Comment:
I actually meant that the delivery strategy has an init method passing Vertx and the ClusterManager which avoid the delivery strategy go pull the cluster manager as it is provided in init method.

Reply:
Yes, see the latest commits, `DeliveryStrategy has been updated too`